src/renderer/page/discover/view.jsx

```diff
@@ -227,10 +227,39 @@ class DiscoverPage extends React.PureComponent {
   }
 
   render() {
-    const { featuredUris, fetchingFeaturedUris } = this.props;
+    const {
+      featuredUris,
+      fetchingFeaturedUris,
+      claimsByChannel,
+      claimsById,
+    } = this.props;
     const hasContent = typeof featuredUris === 'object' && Object.keys(featuredUris).length,
       failedToLoad = !fetchingFeaturedUris && !hasContent;
 
+    if (!!featuredUris && !!claimsByChannel) {
+      let channels = [];
```

Comment:
Can this logic be either moved into `<FeaturedCategory>` and/or moved into a selector that is called by `<FeaturedCategory>`?

Reply:
I moved all this logic into `selectDiscover` and am calling it from the `Discover` page and it's working correctly. Calling it from `<FeaturedCategory>` doesn't make sense to me because the selector returns an object which is mapped to many `<FeaturedCategory>`s in the view.

I can kind of see doing to logic for each category separately but I would need to think about it and it doesn't seem like it will be nearly as clean as this.